cdmtaskservice/mongo.py

```diff
@@ -105,6 +110,17 @@ async def get_image(self, name: str, digest: str | None = None, tag: str | None
             raise NoSuchImageError(f"No image {name} {err} exists in the system.")
         return models.Image.model_construct(**doc)
 
+    async def save_job(self, job: models.Job):
+        """ Save a job. Job IDs are expected to be unique."""
+        _not_falsy(job, "job")
```

Comment:
Do we need this check? I cannot think of a case when `not job` is `true`.

Reply:
If someone does `save_job(None)` or the equivalent. It shouldn't happen, but it ensures the inputs are valid